core-shaded/pom.xml

```diff
@@ -340,7 +312,7 @@
                   <!--
                   1) Don't import packages shaded in the driver bundle. Note that shaded-guava lives
                   in its own bundle, so we must explicitly *not* mention it here.
-                  -->!com.datastax.oss.driver.shaded.netty.*, !com.datastax.oss.driver.shaded.esri.*, !com.datastax.oss.driver.shaded.json.*, !com.datastax.oss.driver.shaded.codehaus.jackson.*, !com.datastax.oss.driver.shaded.fasterxml.jackson.*,
+                  -->!com.datastax.oss.driver.shaded.netty.*, !com.datastax.oss.driver.shaded.json.*, !com.datastax.oss.driver.shaded.codehaus.jackson.*, !com.datastax.oss.driver.shaded.fasterxml.jackson.*,
```

Comment:
I just noticed: we are still mentioning here two shaded packages that do not exist anymore:

* com.datastax.oss.driver.shaded.json
* com.datastax.oss.driver.shaded.codehaus.jackson

Reply:
Ah, good catch, you're absolutely right!  I'll clean this up (and the one below) shortly.